docs/versioned_docs/version-EE/getting-started/license.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Semaphore License
+---
+
+# Get a License
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Available from '@site/src/components/Available';
+import VideoTutorial from '@site/src/components/VideoTutorial';
+import Steps from '@site/src/components/Steps';
+import FeatureNotAvailable from '@site/src/components/FeatureNotAvailable';
+import { NiceButton, ButtonContainer } from '@site/src/components/NiceButton';
+import { GKEIcon, EKSIcon, UbuntuIcon, GCPCompute, AWSEC2Icon } from '@site/src/components/CustomIcons';
+
+This page explains how to obtain a Semaphore Enterprise Edition (EE) license and under which conditions you are required to pay to run your Semaphore EE instance.
+
+## Overview {#overview}
+
+Every Semaphore EE instance must have a valid and current license in order to run. The license is applied during installation and renewed yearly.
+
+## Free Enterprise Edition {#free}
+
+You might not need to pay for the license. If the following conditions apply, you can get a license free and skip the [payment section](#payment):
+
+- You expect to have fewer than 50 users for the Semaphore instance
+- Your company must have less than 5,000,000 US Dollars ARR (Annual Recurring Revenue)
+
+A free licensed Enterprise Edition server includes **the features of a paid installation**. There are no limitations or features missing. The only difference is that a free license **does not include** advanced support or any SLAs. Support is provided on a best effort basis via the [Discord server](https://discord.gg/FBuUrV24NH).
+
+If you do not fulfill these requirements, or if you want advanced support and SLAs, you must [contact support for payment details](#payment) once you install your Semaphore instance.
+
+## Obtain a license {#obtain}
+
+In order to obtain or renew a license, follow these steps:
+
+1. Go to the [Semaphore License Hub](https://licensing.sxmoon.com/)
```

Comment:
@TomFern this is not a license server in production

Reply:
@dexyk thanks for letting me know. What is the correct URL?